tests/test_cli.py

```diff
@@ -100,6 +101,25 @@ def test_run_simple(runner):
     assert result.exit_code == 0
 
 
+def test_workflow(runner):
+    """Test workflow command."""
+    result = runner.invoke(cli.cli, ['run', 'touch', 'data.csv'])
+    assert result.exit_code == 0
+
+    with open('counted.txt', 'w') as stdout:
+        with contextlib.redirect_stdout(stdout):
+            try:
+                cli.cli.main(
+                    args=('run', 'wc', 'data.csv'),
+                    prog_name=runner.get_default_prog_name(cli.cli),
+                )
+            except SystemExit as e:
+                assert e.code in {None, 0}
+
+    result = runner.invoke(cli.cli, ['workflow', 'create', 'counted.txt'])
+    assert result.exit_code == 0
```

Comment:
check that the generated workflow is actually correct?

Reply:
Once I implement reader for workflows and https://github.com/SwissDataScienceCenter/renga-python/pull/93#discussion_r166132578 is solved.